### ISSUE TITLE
Just a suggestion

### ISSUE THREAD

**Issue by sam-thecoder** [satinder147/Image_deionising_auto_encoder#1]
Hi,
Not an ML expert, your code is well formatted and easy to understand but I have a few questions.
First is why you'd use LeakyRelu, I know what it does is prevent vanishing gradient to 0 but I thought there's other optimizers that can do this efficiently, funny enough I can't remember one of head, likely wrong there.
2. You didn't use a deep network with Dense so wouldn't that help make it smarter?
3. What if you could use opencv to add noise and stream a video of rock paper scissors to your autoencoder using time distributed believe that would converge quicker. 
4. You called your model "Models" which is a bit confusing. 
5. You called the auto encoder twice in the training section.

**Comment by satinder147** [comment]
Hello sir, 
1)I used relu with adam optimizer at first. Although I was getting good accuracy with that as well but I though it would be even better if I used LeakyRelu, because it takes care of the activation which become less than 0.
2)Sir I think even it would have worked fine. But adding a dense layer of 240X240X3 will increase the parameters exponentially
3)I think it is a good idea, I will try implementing it
4)I will fix that
5)It is a typo, I will fix that

Thanks for the suggestions. Do check out my other repos as well. They are equally good.
